### PR TITLE
Color theme fixes

### LINKED ISSUE #882: "Friend typing" is now written in black instead of grey
Since the last update -- that is to say after the pull request #869 made by @muchweb and @GrayHatter has been merged -- the "friend typing" notification is now black instead of grey.

Here is a screenshot, Ubuntu 14.04

friend_typing

### PR DESCRIPTION
**Pull request by tsudoko**
Fixes #882, makes the default theme look more like it looked like before the colours branch was merged.
Fixes some major graphical issues (e.g. file transfers, default avatars) with other themes as well

### CODE REVIEW

File: messages.c [hunk 1]
```diff
@@ -137,11 +137,7 @@ void messages_draw(MESSAGES *m, int x, int y, int width, int height)
                 h2 = STRING_IDX_MAX;
             }
 
-            if(msg->author) {
-                setcolor(COLOR_MAIN_SUBTEXT);
-            } else {
-                setcolor(COLOR_MAIN_TEXT);
-            }
+            setcolor(COLOR_MAIN_CHATTEXT);
```
Comment (GrayHatter) [inline messages.c]: why

Comment (tsudoko) [inline messages.c]: That's how chat text looked like before the merge. If it bothers anyone I can restore it.

Comment (muchweb) [inline messages.c]: It was actually an improvement, to have author messages in different colours.

**Comment by muchweb** [comment]
This PR will also fix #881

**Comment by GrayHatter** [comment]
@muchweb #881 is already fixed in master, sorry I forgot to reference that issue in my commit.

**Pull request merged**